### PR TITLE
build container with external repos

### PR DESCRIPTION
**Pull request by mcalmer**

### CODE REVIEW

File: docker/salt-sle12sp1/add_repositories.sh [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-zypper ar -f http://download.suse.de/ibs/SUSE:/SLE-12-SP1:/Update:/Products/standard "SLE12 SP1 Update Products"
-zypper ar -f http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/testing/SLE_12/ "salt"
+zypper ar -f http://nu.novell.com/SUSE/Products/SLE-SERVER/12-SP1/x86_64/product/ "SLES 12 SP1 Pool"
+zypper ar -f http://nu.novell.com/SUSE/Updates/SLE-SERVER/12-SP1/x86_64/update/ "SLES 12 SP1 Updates"
+zypper ar -f http://nu.novell.com/SUSE/Products/SLE-SDK/12-SP1/x86_64/product/ "SLE-SDK 12 SP1 Pool"
+zypper ar -f http://nu.novell.com/SUSE/Updates/SLE-SDK/12-SP1/x86_64/update/ "SLE-SDK 12 SP1 Updates"
+zypper ar -f http://nu.novell.com/SUSE/Products/SLE-Module-Public-Cloud/12/x86_64/product/ "SLE-Module-Public-Cloud 12 Pool"
+zypper ar -f http://nu.novell.com/SUSE/Updates/SLE-Module-Public-Cloud/12/x86_64/update/ "SLE-Module-Public-Cloud 12 Updates"
+
+zypper ar -f http://download.opensuse.org/repositories/systemsmanagement:/saltstack/SLE_12/ "salt"
```
Comment (isbm) [inline docker/salt-sle12sp1/add_repositories.sh]: Maybe I am missing something, but shouldn't it be a production already? We probably should not touch anything from this repo during we are checking if `testing` is fine. Or?

Comment (mcalmer) [inline docker/salt-sle12sp1/add_repositories.sh]: Well, we need the other packages (python-*) from it to solve the dependencies of salt.
To make sure, that we are really using the salt package from testing I changed the priority of the salt_testing repo.

To be sure, I will try to create an image without this, but I think there are some packages included we do not have yet in SLE12 or SLE11.

Comment (mcalmer) [inline docker/salt-sle12sp1/add_repositories.sh]: Package 'python-salt-testing' not found.
python-msgpack-python > 0.3 needed by salt-2015.8.7-15.1.x86_64

So we need the salt repo as long as we do not have these packages somewhere else

Comment (isbm) [inline docker/salt-sle12sp1/add_repositories.sh]: Copy them _from_ there to the testing, perhaps?

**Comment by mcalmer** [comment]
I used nu.novell.com, because it is accessible from external using basic auth with
username and password (mirror credentials)

File: docker/salt-sle12sp1/add_repositories.sh (hunk 1 above)
Comment (mcalmer) [inline docker/salt-sle12sp1/add_repositories.sh]: As soon as we have released SUSE Manager, we should use SUSE Manager repos here.

**Comment by isbm** [comment]
Actually, `nu.novell` is a very good idea. Maybe you could add a comment there for external users regarding the authentication?

**Pull request merged**